crates/astria-composer/src/executor/mod.rs

```diff
@@ -493,14 +506,22 @@ async fn get_latest_nonce(
             },
         );
     let res = tryhard::retry_fn(|| {
-        let client = client.clone();
-        let span = info_span!(parent: span.clone(), "attempt get nonce");
+        let mut client = client.clone();
```

Comment:
why is this mut

Reply:
not sure, but the generate code that it's from needs it mut